Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[requires]
+python_version = "2.7"
+
+[dev-packages]
+
+
+
+[packages]
+
+"flake8" = "*"
+"flake8-isort" = "*"
+funcsigs = "*"
+mozlog = "*"
+PyPOM = "*"
+pytest = "*"
+pytest-metadata = "*"
+pytest-selenium = "*"
+pytest-variables = "*"
+pytest-xdist = "*"
+"querystringsafe_base64" = "==1.0.0"
```

Comment:
Strange that this is in quotes. Is it because of the `_` in the name?

Reply:
I believe so - was done automagically via pipenv install